src/code_lens_provider/virtualSqlCodeLensProvider.ts

```diff
@@ -35,7 +35,7 @@ export class VirtualSqlCodeLensProvider implements CodeLensProvider {
   ): CodeLens[] | Thenable<CodeLens[]> {
     // Enable this code lens only for adhoc query files created using command: dbtPowerUser.createSqlFile
     if (
-      document.uri.scheme !== "untitled" &&
+      document.uri.scheme !== "untitled" ||
       document.languageId !== "jinja-sql"
```

Comment:
untitled is our scheme or coming from vscode?

Reply:
untitled is vscode scheme for new files